app/views/passwords/edit.haml

```diff
@@ -1,27 +1,29 @@
 - content_for :page_title do
-  = "#{AppConfig.settings.pod_name} / #{t('devise.passwords.edit.change_password')}"
+  = "#{AppConfig.settings.pod_name} - #{t('devise.passwords.edit.change_password')}"
 
 #reset_password
-  %h1{id: "huge-text"}= AppConfig.settings.pod_name
-
-  = form_for(resource, :as => resource_name, :url => password_path(resource_name), :html => {:class => "form-horizontal block-form", :method => :put }, :autocomplete => 'off') do |f|
-    %fieldset
-      = f.hidden_field :reset_password_token
-      .control-group
-        %label{class: "control-label", for: "user_password", style: "width: 45%; text-align: right; padding: 10px 0; overflow: hidden"}
+  .container-fluid
+    .text-center
+      .branding-asterisk
+      %h1
+        = AppConfig.settings.pod_name
+          
```

Comment:
They are still there. Please remove them and also check for trailing whitespaces in other files you changed.